### ISSUE TITLE
Crashes in Content Processes starting in Firefox 111

### ISSUE THREAD

**Issue by tomrittervg** [xiaoxiaoflood/firefox-scripts#241]
Users of Nightly 111 may have started experiencing content process crashes in the past 2 days. Once 111 goes to Beta/Release, users on Beta or Release may start experiencing incorrect behavior.

This is because I landed [1811294](https://bugzilla.mozilla.org/show_bug.cgi?id=1811294) which will cause content process crashes when dynamically named string preferences (i.e. preferences that aren't specified in prefs.js or StaticPrefList) are accessed in the content process.  I'm here because we just got crash reports for one user accessing `userChrome.searchSelectionShortcut.keycode` which I know is a userchrome thing.

There are preferences that affect the crashing behavior referenced in that bug. As before, Mozilla's policy is that userschrome stuff is not supported, so we're not going to go out of our way to make things work* and we don't recommend you disable security settings.  But at the same time we appreciate all of our users and as one hacker to another(s) - I wanted to give you a head's up.

\* well actually, we've been going really far out of our way for a while, but there are limits

**Comment by tomrittervg** [comment]
No; the crashing behavior was only turned on in 111 a couple days ago.

**Comment by xiaoxiaoflood** [comment]
Thank you for reaching out.

I'm on v111 with all extensions and scripts maintained by me and no issues so far, I guess that change doesn't affect the content of this repo, but anyway is good to know.
